java-squid/src/main/java/org/sonar/java/cfg/CFG.java

```diff
@@ -563,9 +548,12 @@ private void buildConditionalOr(BinaryExpressionTree tree) {
   }
 
   private void buildLabeledStatement(LabeledStatementTree labeledStatement) {
+    String name = labeledStatement.label().name();
+    labelsBreakTarget.put(name, currentBlock);
+    pendingLabel = name;
     build(labeledStatement.statement());
     currentBlock = createBlock(currentBlock);
-    labels.put(labeledStatement.label().name(), currentBlock);
+//    labels.put(labeledStatement.label().name(), currentBlock);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3ACommentedOutCodeLine)